src/components/LegacyTooltip/LegacyTooltip.tsx

```diff
@@ -404,8 +407,9 @@ export const LegacyTooltip = ({
                             style={popperInstance.styles.arrow}
                             className={merge([
                                 withArrow &&
-                                    'tw-popper-arrow tw-z-[110000] tw-absolute tw-w-3 tw-h-3 tw-pointer-events-none before:tw-absolute before:tw-w-3 before:tw-h-3 before:tw-rotate-45 before:tw-border before:tw-border-line',
+                                    'tw-popper-arrow tw-z-[110000] tw-absolute tw-w-3 tw-h-3 tw-pointer-events-none before:tw-absolute before:tw-w-3 before:tw-h-3 before:tw-rotate-45 before:tw-border',
```

Comment:
The color of the border moved a couple lines down as now it can be changed based on the prop.

Reply:
But the previous border only rendered when `withArrow` was `true`.  This will apply the border by default now.  Maybe keep the `before:tw-border-line` in the `withArrow` condition and the `withStrongBorder` check will overwrite it if applied?